### PR TITLE
chore: final mint step and share

### PR DESCRIPTION
**Pull request by csosamorales**

### CODE REVIEW

File: ui/src/views/home/nfa-list/nfa-list.tsx [hunk 1]
```diff
@@ -65,10 +65,13 @@ export const NFAList = () => {
           Next page
         </Button>
       </Flex>
-      <Flex css={{ gap: '$2' }}>
+      <div>
```
Comment (zoruka) [inline ui/src/views/home/nfa-list/nfa-list.tsx:68]: seems like this `div` may be removed?

File: ui/src/views/mint/mint.context.tsx [hunk 2]
```diff
@@ -114,8 +119,15 @@ export abstract class Mint {
           config={{
             transaction: {
               onSuccess: (data) => {
-                console.log('Successfully minted! what now?', data);
-                alert('transaction hash: ' + data.transactionHash);
+                // AppLog.info('Transaction hash: ' + data);
+                pushToast('success', 'Successfully minted!', {
+                  onDismiss: () => setMintSuccess(true),
+                  duration: 2000,
+                });
```
Comment (zoruka) [inline ui/src/views/mint/mint.context.tsx:126]: I think you don't need the toast dismiss to show up the success string, both may be triggered at the same time so user do not need to wait these 2 seconds to interact.

File: ui/src/views/mint/mint.context.tsx [hunk 3]
```diff
@@ -114,8 +119,15 @@ export abstract class Mint {
           config={{
             transaction: {
               onSuccess: (data) => {
-                console.log('Successfully minted! what now?', data);
-                alert('transaction hash: ' + data.transactionHash);
+                // AppLog.info('Transaction hash: ' + data);
```
[the remark this answers is not in the document]
Comment (csosamorales) [reply ui/src/views/mint/mint.context.tsx:122]: I'll remove it once we merge the console.log PR, but I can add a TODO comment

File: ui/src/views/home/nfa-list/nfa-list.tsx (hunk 1 above)
Comment (csosamorales) [reply ui/src/views/home/nfa-list/nfa-list.tsx:68]: this view will def change once we have the designs, but for now I need the div to display the items like this
image

File: ui/src/views/mint/mint-stepper.tsx [hunk 4]
```diff
@@ -4,35 +4,47 @@ import { GithubStep } from './github-step';
...
+  const {
+    transaction: { isSuccess },
+  } = Mint.useTransactionContext();
 
-        <Stepper.Step>
-          <MintStep header="Connect GitHub and select repository">
-            <GithubStep />
-          </MintStep>
-        </Stepper.Step>
+  console.log('isSuccess', isSuccess);
```
Comment (zoruka) [inline ui/src/views/mint/mint-stepper.tsx:15]:
```suggestion
```

File: ui/src/views/mint/mint.context.tsx [hunk 5]
```diff
@@ -118,8 +118,10 @@ export abstract class Mint {
           config={{
             transaction: {
               onSuccess: (data) => {
-                AppLog.successToast('Successfully minted!');
-                alert('transaction hash: ' + data.transactionHash);
+                AppLog.info('Transaction hash: ' + data);
```
Comment (zoruka) [inline ui/src/views/mint/mint.context.tsx:121]:
```suggestion
                AppLog.info('Transaction:', data);
```

**Pull request merged**